test_scripts/API/Expanded_proprietary_data_exchange/018_OnSystemRequest_with_OEM_SPECIFIC_requestType_binaryData.lua

```diff
@@ -0,0 +1,60 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0083-Expandable-design-for-proprietary-data-exchange.md
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/2716
+--
+-- In case:
+-- 1. HMI sends BC.OnSystemRequest with `requestType` = OEM_SPECIFIC and not empty `fileName` parameters
+--
+-- SDL does:
+-- 1. Read content of the file provided in `fileName`
+-- 2. Send OnSystemRequest to mobile App with content of the file in `binaryData`
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/Expanded_proprietary_data_exchange/commonDataExchange')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local content = "ABC"
```

Comment:
Might want to use something like a key-value pair for the binaryData content (as mentioned in the original issue)
```suggestion
local content = "{\"key\":\"value\"}"
```
or test with multiple content values